src/plugins/intel_cpu/src/nodes/mkldnn_conv_node.cpp

```diff
@@ -559,17 +559,31 @@ bool MKLDNNConvolutionNode::created() const {
     return getType() == Convolution;
 }
 
+// start declaration to avoid -Wmissing-declarations
 std::shared_ptr<mkldnn::convolution_forward::desc>
-MKLDNNConvolutionNode::createDescriptorInternal(const mkldnn::memory::desc& inputDesc,
-                                                const mkldnn::memory::desc& weightDesc,
-                                                const mkldnn::memory::desc& biasDesc,
-                                                const mkldnn::memory::desc& outputDesc,
-                                                bool withBiases,
-                                                const std::vector<size_t>& stride,
-                                                const std::vector<ptrdiff_t>& dilation,
-                                                const std::vector<ptrdiff_t>& paddingL,
-                                                const std::vector<ptrdiff_t>& paddingR,
-                                                mkldnn::algorithm alg) {
+createDescriptorInternal(const mkldnn::memory::desc& inputDesc,
+                         const mkldnn::memory::desc& weightDesc,
+                         const mkldnn::memory::desc& biasDesc,
+                         const mkldnn::memory::desc& outputDesc,
+                         bool withBiases,
+                         const std::vector<size_t>& stride,
+                         const std::vector<ptrdiff_t>& dilation,
+                         const std::vector<ptrdiff_t>& paddingL,
+                         const std::vector<ptrdiff_t>& paddingR,
+                         mkldnn::algorithm alg);
+// end declaration to avoid -Wmissing-declarations
+
+std::shared_ptr<mkldnn::convolution_forward::desc>
+createDescriptorInternal(const mkldnn::memory::desc& inputDesc,
+                         const mkldnn::memory::desc& weightDesc,
+                         const mkldnn::memory::desc& biasDesc,
+                         const mkldnn::memory::desc& outputDesc,
+                         bool withBiases,
+                         const std::vector<size_t>& stride,
+                         const std::vector<ptrdiff_t>& dilation,
+                         const std::vector<ptrdiff_t>& paddingL,
+                         const std::vector<ptrdiff_t>& paddingR,
+                         mkldnn::algorithm alg) {
```

Comment:
Either mark this function static or move it to the anonymous namespace to make it immanent to this translation unit and  avoid any possible name conflicts.  In that case you probably do not need the declaration at all.